docs/hopsml/provenance.rst

```diff
@@ -0,0 +1,104 @@
+Provenance
+==========
+The provenance layer in Hopsworks allows for the tracking of file(HopsFS) and application(Yarn) operations in order to provide users with aditional details.
+There are currently four types of provenance:
+
+* DISABLED
+* META - enables search of projects/datasets.
+* MIN - enables search of featuregroups/training datasets/features including attached tags.
+* FULL - allows linking of feature groups/training datasets/experiments/models.
+
+**Note:** Linking of ML artifacts (FULL provenance) is a Hopsworks Enterprise feature.
+
+Temporary provisions (These will be updated soon)
```

Comment:
What do you mean?

Reply:
updated to `Temporary limitations` - so that people don't come to me and be like why isn't this working like this... like Why can't I change provenance of a project. There are JIRAs created to deal with these limitations... and when they are done these limitations will be lifted.